openmp/docs/atomic/data.json

```diff
@@ -4,44 +4,46 @@
     "Name": "atomic",
     "DirectoryName": "atomic",
     "SharedDescription": true,
-    "Description": "The reduction clause indicates that the variables passed are, as its name suggests, used in a reduction. Each implicit task or SIMD lane creates a private copy initialises it to the initialiser value of the reduction identifier, that is, 0 for a sum or 1 for a product to name a few. After the end of the region, the original list item is updated with the values of the private copies using the combiner associated with the reduction identifier.\nBy default, the reduction computation is complete at the end of the construct. However, if nowait is specified on the construct, this is no longer guaranteed. Indeed, accesses to the original list item will create a data race and, thus, have unspecified effect unless synchronisation ensures that they occur after all threads have executed all of their iterations or section constructs, and the reduction computation has completed and stored the computed value of that list item. This can most simply be ensured through a barrier synchronisation.",
+    "Description": "The atomic construct ensures that actions performed on a memory location are only done by a single thread. It prevents the possibility of multiple, simultaneous read / write actions.  It primarily protects a single update to a shared variable and only applies to a single statement. \n  An important distinction of atomic is that it protects the memory locations for which actions are performed; multiple atomic statements can be run together as long as they do not work on the same memory.",
```

Comment:
"The atomic construct ensures that actions performed on a memory location are only done by a single thread." What about "The `atomic` construct ensures that a given action performed on a memory location cannot be interleaved with other operations."
The other definition seems to echo the `single` construct.